### PR TITLE
Feat/video and image on article

### PR DESCRIPTION
**Pull request by aethiss**
## Description: Improvement 
We Add a new dynamic logic inside the Article component in order to render a (inline) video, or image teaser or image default.
Also we add test and defensive code.

## Known Issues: the known issue here if any
we need add style (will discuss on next stand-up)
the logic to create a video SRC is hardcoded inside the component, in the future we should receive from DSP.

### Ticket: the ticket related to this PR
https://applicaster.atlassian.net/browse/MS1-279

### Checklist for this pull request

- [x] PR is scoped to one task
- [x] Tests included

One of:
  * [] The PR is not making any UI change
  * [x] The PR is making a UI change but not a product change
    * [x] Screenshots included

Screenshot 2019-11-22 at 15 55 36

### CODE REVIEW

File: src/components/Article.js [hunk 1]
```diff
@@ -72,29 +67,62 @@ class Article extends Component {
...
-    const main_image =
-      R.path(['media_group', '0', 'media_item', '0'], data) || {};
 
     const {
       RN_Article_Background,
       RN_Article_Shadow,
       ArticlePaddingHorizontal,
     } = styles;
 
+    // console.log('PROPS');
```
Comment (giorgiomazzei79) [inline src/components/Article.js]: I would delete these comments

File: src/components/Article.js [hunk 2]
```diff
@@ -1,14 +1,9 @@
+/* eslint-disable quote-props */
```
Comment (giorgiomazzei79) [inline src/components/Article.js]: Could you explain why we need to disable this rule please?

File: src/components/__tests__/Article.spec.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import 'react-native';
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import Article from '../Article';
+import * as mockPropsVideo from './moks/ArticlePropsVideo.json';
```
Comment (giorgiomazzei79) [inline src/components/__tests__/Article.spec.js]: There's a typo in the path. It should be './mocks/ArticlePropsVideo.json'

File: src/components/Article.js (hunk 1 above)
Comment (aethiss) [reply src/components/Article.js]: done

File: src/components/Article.js (hunk 2 above)
Comment (aethiss) [reply src/components/Article.js]: removed, was added automatically from the editor.

File: src/components/__tests__/Article.spec.js (hunk 3 above)
Comment (aethiss) [reply src/components/__tests__/Article.spec.js]: done

**Pull request merged**